src/pages/Order/components/Input.tsx

```diff
@@ -0,0 +1,39 @@
+import styled from "@emotion/styled";
+import type { ComponentPropsWithoutRef } from "react";
+import ErrorMsg from "@/pages/Order/components/ErrorMsg";
+
+interface InputProps extends ComponentPropsWithoutRef<"input"> {
+  errorMsg: string | null;
+}
+
+const Input = ({ type, placeholder, value, onChange, errorMsg }: InputProps) => {
+  return (
+    <Wrapper>
+      <Content type={type} placeholder={placeholder} value={value} onChange={onChange} errorMsg={errorMsg} />
```

Comment:
```suggestion
const Input = ({ type, placeholder, value, onChange, errorMsg, ...props }: InputProps) => {
  return (
    <Wrapper>
      <Content type={type} placeholder={placeholder} value={value} onChange={onChange} errorMsg={errorMsg} {...props} />
```

이렇게 다른 props도 수용할 수 있는 형태면 좋을 것 같네요!